# Практическая работа №2: Обработка выборочных данных. Нахождение точечных оценок параметров распределения.

Выполнили студентки гр. 0382 Деткова Анна и Рубежова Наталия. Вариант №10

## Цель работы

Получение практических навыков нахождения точечных статистических оценок параметров распределения.

## Основные теоретические положения

**Условные варианты** - варианты, определяемые равенством: $$u_i = \frac {x_i - C} h$$, где C - условный ноль, значение которое выбирается равным значению варианты интервального ряда, являющуюся средней или близкой к средней по значению в этом ряду. В результате все варианты оказываются целыми числами. $x_i$ - наблюдаемая варианта. $h$ - шаг.

Выполняется следующее равенство:
$$\sum n_i (u_i+1)^4 = \sum n_i u_i^4 + 4 \sum n_i u_i^3 + 6 \sum n_i u_i^2 + 4 \sum n_i u_i + \sum n_i$$

---

**Обычный эмпирический момент порядка $k$** - среднее значение $k$-ых степеней разностей $x_i-C$: $$M_k' = \frac 1 N \sum n_i (x_i-C)^k$$, $N$ - размер выборки, $n_i$ - частота варианты.

**Начальный эмпирический момент порядка k** - обычный момент порядка $k$ при $С = 0$: $$M_k = \frac 1 N \sum n_i x_i^k$$

В частности **выборочное среднее** - начальный эмпирический момент первого порядка: $$M_1 = \frac 1 N \sum n_i x_i = \overline {x_в}$$

**Центральный эмпирический момент порядка $k$** - обычный момент порядка $k$ при $С = \overline {x_в}$: $$m_k = \frac 1 N \sum n_i (x_i-\overline {x_в})^k$$

В частности **выборочная дисперсия** - центральный эмпирический момент второго порядка: $$m_2 = \frac 1 N \sum n_i (x_i-\overline {x_в})^2$$

Причем справедливы соотношения:

$m_2 = M_2' - (M_1')^2$,

$m_3 = M_3' - 3M_2'M_1' + 2(M_1')^3$,

$m_4 = M_4' - 6M_3'M_1' + 6M_2'(M_1')^2 - 3(M_1')^4$

---

**Условный эмпирический момент порядка $k$** - начальный момент порядка $k$, вычисленный для условных вариант: $$M_k^* = \frac 1 N \sum n_i u_i^k = \frac 1 N \sum n_i (\frac {x_i - C} h )^k$$

В частности $$M_1^* = \frac 1 N \sum n_i u_i^k = \frac 1 N \sum n_i |\frac {x_i - C} h| = \frac 1 h (\frac 1 N \sum n_i x_i - C \frac 1 N \sum {n_i}) = \frac 1 h (\overline {x_в} - C)$$

Отсюда $\overline {x_в} = M_1^* h + C$

Обычные моменты через условные: $M_k^* = \frac 1 {h^k} \frac 1 N n_i(x_i - C)^k = \frac {M_k'} {h^k}$, отсюда $M_k' = M_k^*h^k$.

Также можно получить центральные выборочные моменты, выражающие центральные моменты через условные:

$m_2^* = (M_2^* - (M_1^*)^2)h^2$,

$m_3^* = (M_3^* - 3M_2^*M_1^* + 2(M_1^*)^3)h^3$,

$m_4^* = (M_4^* - 6M_3^*M_1^* + 6M_2^*(M_1^*)^2 - 3(M_1^*)^4)k^4$

<br>


---
Для вычисления **выборочного среднего с помощью стандартной формулы** используем формулу: $\overline{x_в} = \frac 1 N \sum n_i x_i$, где $x_i$ - варианта.

Для вычисления **выборочного среднего с помощью условных вариант** используем формулу: $\overline {x_в} = M_1^*h + C$.

Для вычисления **выборочной дисперсии с помощью стандартной формулы** используем формулу: $\sigma_в^2 = \frac 1 N \sum n_i(x_i - \overline{x})^2$, где $x_i$ - варианта.

Для вычисления **выборочного среднего с помощью условных вариант** используем формулу: $\sigma_в^2 = (M_2^*-(M_1^*)^2)h^2$.

Для вычисления **исправленной выборочной дисперсии** используется следующая формула: $S^2 = \frac {N} {N-1} \sigma_в$.

Для вычисления **исправленного выборочного СКО** используется следующая формула: $S = \sqrt {S^2}$.
Для вычисления **выборочного СКО** используется следующая формула: $\sigma_в = \sqrt {\sigma^2_в}$.

---

Для вычисления **статистической оценки коэффициента ассиметрии** используется формула: $a_s^* = \frac {m_3^*} {s^3}$.  Асимметрия характеризует меру скошенности полигона или гистограммы влево / вправо относительно самого высокого участка.<br>
Правосторонняя асимметрия характеризуется удлинённым правым «хвостом». Левостороння асимметрия, наоборот, характеризуются удлинённым левый «хвостом».<br>
Смысл:
* если $a_s > 0$, то распределение скошено вправо, если $a_s < 0$ – то влево;
* принята следующая условная градация: если $|a_s| < 0.25$, то асимметрия незначительна, если $0.25 < |a_s| < 0.5$, то умеренная, и если $|a_s| > 0.5$, то существенная.

Для вычисления **статистической оценки коэффициента эксцессы** используется формула: $\epsilon_s^* = \frac {m_4^*} {s^4} - 3$. Этот коэффициент характеризует высоту и пологость полигона или гистограммы вверх / вниз.<br>
Смысл:
* если $\epsilon_s^* > 0$, то эмпирическое распределение является более высоким («островершинным») – относительно «эталонного» нормального распределения с параметрами $a = \overline {x_в}, \sigma = \sigma_в$. Если же  $\epsilon_s^* < 0$ – то более низким и пологим;
* чем больше $\epsilon_s^*$ по модулю, тем «аномальнее» высота в ту или иную сторону.

---

**Мода** — это значение, встречающееся в выборке чаще других.<br>
Мода для интервального ряда вычисляется по следующей формуле:
$$M_o^* = x_0 + \frac {n_M - n_{M-1}} {(n_M - n_{M-1}) + (n_M - n_{M+1})}h$$
где $x_0$ - нижняя граница модального интервала (интервал с наибольшей частотой), $n_M$ - частота модального интервала, $n_{M-1}$ - частота интервала предыдущего до модального, $n_{M+1}$ - частота интервала следующего после модального.<br>


**Медиана** — это число, которое окажется строго по центру списка чисел в наборе данных, если их предварительно упорядочить по возрастанию.<br>
Медиана для интервального ряда вычисляется по следующей формуле:
$$M_e^* = x_0 + \frac {0.5N - n^k_{m-1}} {n_m} h$$
где $N$ - объем выборки, $n_m$ - частота медианного интервала, $n^k_{m-1}$ - накопленная частота предыдущего интервала, $x_0$ - нижняя граница медианного интервала.<br>

**коэффициент вариации** – это отношение стандартного отклонения к средней, выраженное в процентах.<br>
Коэффициент вариации вычисляется по формуле:
$$V^* = \frac {\sigma_в} {\overline {x_в}} 100\%$$

* если показатель вариации составляет примерно 30% и меньше, то статистическая совокупность считается однородной. Это означает, что большинство вариант находится недалеко от средней, и найденное значение $\overline {x_в}$ хорошо характеризует центральную тенденцию совокупности.

* если показатель вариации составляет существенно больше 30%, то совокупность неоднородна, то есть, значительное количество вариант находятся далеко от $\overline {x_в}$, и выборочная средняя плохо характеризует типичную варианту.

## Постановка задачи

Для заданных выборочных данных вычислить с использованием метода моментов и условных вариант точечные статистические оценки математического ожидания, дисперсии, среднеквадратичного отклонения, асимметрии, эксцесса, моды, медианы и коэффициента вариации исследуемой случайной величины. Полученные результаты содержательно проинтерпретировать.

## Выполнение работы

По возможности каждый пункт работы выполняется с помощью кода. Рекомендуемые языки программирования для выполнения работ - R и Python.

In [240]:
# Подключение библиотек
import pandas
import numpy as np
import matplotlib.pyplot as plt
import math

### 1. Для середин интервального ряда, полученного в практической работе №1, вычислить условные варианты. Заполнить табл. 1 (в последней строке $\sum$ необходимо заполнить суммы столбцов; ячейки отмеченные прочерком заполнять не надо). Провести контроль вычислений.

Загрузка генеральной совокупности и формирование выборки простым бесповторным отбором.

In [241]:
# Загрузка генеральной совокупности из csv-формата в тип данных Data Frame
general_df = pandas.read_csv('sample.csv', comment='#')

# Заданный объем выборки
N = 117

# Формирование выборки. Способ формирования: простой бесповторный отбор
sample_df = general_df.sample(n=N, replace=False)

Построение интервального ряда, аналогично заданию 4 лаб.1.

In [242]:
# Ранжированный ряд по признаку nu
ranked_df_nu = sample_df.sort_values(by='nu')
# Добавление колонки с рангом
ranked_df_nu['rank']=list(range(0, sample_df.shape[0]))

# Вычисление наибольшего и наименьшего значения в выборке
x_max_nu = max(ranked_df_nu['nu'])
x_min_nu = min(ranked_df_nu['nu'])

# Вариационный ряд из ранжированного ряда по признаку nu
variation_df_nu = ranked_df_nu.drop_duplicates(subset='nu', keep='first')

# Получение интервального ряда
# Объем выборки
N=117
# Количество интервалов по формуле Стерджесса
k = int(np.floor(1 + 3.31 * np.log10(N)))

# Ширина интервалов h
h = int(np.ceil((x_max_nu - x_min_nu) / k))

# Границы интервалов
# Вставим левую границу для i=1 первого интервала
left = x_min_nu + (1-1)*h
left_borders = [left]
right_borders = []
# Вставим в списки итеративно следующие границы
for i in range(1, k):
  left_borders.append(x_min_nu + h * i)
  right_borders.append(x_min_nu + h * i)
# Добавим последнюю правую границу
right = x_min_nu + k*h
right_borders.append(right)
# Разобьем на интервалы
intervals= []
for i in range(len(left_borders)):
  intervals.append((left_borders[i], right_borders[i]))

# Вычисляем середины интервалов
mid_of_intervals = []
for i in range(len(intervals)):
  left, right = intervals[i]
  mid_of_intervals.append((left + right) / 2)

# Абсолютные частоты для интервалов
abs_freq = [0 for i in range(0, k)]
# Считаем частоту попаданий вариант в каждый интервал
for varianta in variation_df_nu['nu']:
  for i in range(len(intervals)):
    left, right = intervals[i]
    if varianta < right and varianta >= left:
      abs_freq[i]+=1
# Количество вариант
variant_count = variation_df_nu['nu'].shape[0]
# Относительные частоты для интервалов
rel_freq = [freq/ variant_count for freq in abs_freq]

# Накопленные частоты для интервалов
accum_abs_freq = []
accum_rel_freq = []
for i in range(len(abs_freq)):
  accum_abs_freq.append(np.sum(abs_freq[:i+1]))
  accum_rel_freq.append(np.sum(rel_freq[:i+1]))

# Добавляем необходимые колонки для Таблицы 1
interval_df = pandas.DataFrame({'i': list(range(1, k+1))})
interval_df["[x_i, x_i+1)"] = intervals
interval_df['Середина i интервала'] = mid_of_intervals
interval_df['n_i'] = abs_freq
interval_df['p*_i'] = rel_freq
interval_df['Накопленная n_i'] = accum_abs_freq
interval_df['Накопленная p*_i'] = accum_rel_freq

interval_df

i [x_i, x_i+1)  Середина i интервала  n_i      p*_i  Накопленная n_i  \
0  1   (321, 360)                 340.5    6  0.065934                6   
1  2   (360, 399)                 379.5   13  0.142857               19   
2  3   (399, 438)                 418.5   19  0.208791               38   
3  4   (438, 477)                 457.5   21  0.230769               59   
4  5   (477, 516)                 496.5   15  0.164835               74   
5  6   (516, 555)                 535.5   14  0.153846               88   
6  7   (555, 594)                 574.5    3  0.032967               91   

   Накопленная p*_i  
0          0.065934  
1          0.208791  
2          0.417582  
3          0.648352  
4          0.813187  
5          0.967033  
6          1.000000

Построение таблицы с вычисленными условными вариантами.

In [243]:
# нахождение условного нуля С - середина интервала,
# который находится по середине (округление вверх) интервального ряда - 4 интервал
C = interval_df["Середина i интервала"][round(len(interval_df) / 2)]

# построение таблицы 1
cond_var_df = pandas.DataFrame({'i': list(range(1, k+1))})
cond_var_df["x"] = mid_of_intervals
cond_var_df["n"] = abs_freq
cond_var_df["u"] = (mid_of_intervals - C) / h
cond_var_df["n*u"] = cond_var_df["n"] * cond_var_df["u"]
cond_var_df["n*u2"] = cond_var_df["n"] * pow(cond_var_df["u"], 2)
cond_var_df["n*u3"] = cond_var_df["n"] * pow(cond_var_df["u"], 3)
cond_var_df["n*u4"] = cond_var_df["n"] * pow(cond_var_df["u"], 4)
cond_var_df["n*(u+1)4"] = cond_var_df["n"] * pow(cond_var_df["u"] + 1, 4)

# рассчет сумм каждого столбца
sums = ["-", cond_var_df["x"].sum(), cond_var_df["n"].sum(), "-",
        cond_var_df["n*u"].sum(), cond_var_df["n*u2"].sum(),
        cond_var_df["n*u3"].sum(), cond_var_df["n*u4"].sum(),
        cond_var_df["n*(u+1)4"].sum()]

#  добавление сумм вниз таблицы
cond_var = pandas.concat([cond_var_df,
                         pandas.DataFrame(data=[sums],
                                          columns=["i", "x", "n", "u", "n*u", "n*u2", "n*u3", "n*u4", "n*(u+1)4"])],
                          ignore_index=True)
cond_var

print('Таблица 1:')
# Отображение Таблицы 1
cond_var.style.set_properties(**{'text-align': 'center'}).hide(axis='index')

Таблица 1:


Проведение контроля вычислений по формуле для абсолютных частот: $\sum n_i (u_i+1)^4 = \sum n_i u_i^4 + 4 \sum n_i u_i^3 + 6 \sum n_i u_i^2 + 4 \sum n_i u_i + \sum n_i$.

In [244]:
# сравнение левой и правой части формулы
sum = sums[7] + 4 * sums[6] + 6 * sums[5] + 4 * sums[4] + sums[2]
print(f"По итогу вычисленной суммы по формуле было получено значение {sum}; сравним полученные результаты: {np.abs(sum - sums[8]) < 0.00000001}")

По итогу вычисленной суммы по формуле было получено значение 1195.0; сравним полученные результаты: True


### 2. Вычислить условные эмпирические моменты $\nu_i^*$ через условные варианты. С помощью условных эмпирических моментов вычислить центральные эмпирические моменты $\mu_i^*$. Полученные результаты занести в табл. 2.

Для условных эмпирических моментов (1, 2, 3 и 4 порядков) используется следующая формула:
$\nu_k^* = \frac 1 N \sum n_i u_i^k$, где $n_i$ - абсолютная частота.

Для центральных эмпирических моментов используется следующая формула: $\mu_k^*=M(X-M(X))^k$, причем справедливы соотношения:

$\mu_2^*=(\nu_2^*-(\nu_1^*)^2)h^2$,

$\mu_3^*=(\nu_3^*-3\nu_2^*\nu_1^* + 2(\nu_1^*)^3)h^3$,

$\mu_4^*=(\nu_4^*-6\nu_3^*\nu_1^* + 6\nu_2^*(\nu_1^*)^2 - 3(\nu_1^*)^4)h^4$

In [245]:
# построение таблицы 2
moments_df = pandas.DataFrame({'i': list(range(1, 5))})
moments_df["nu"] = [sums[4] / N, sums[5] / N, sums[6] / N, sums[7] / N]
moments_df["mu"] = ["-",
                    (moments_df["nu"][1] - pow(moments_df["nu"][0], 2)) * pow(h, 2),
                    (moments_df["nu"][2] - 3 * moments_df["nu"][1] * moments_df["nu"][0] + 2 * pow(moments_df["nu"][0], 3)) * pow(h, 3),
                    (moments_df["nu"][3] - 6 * moments_df["nu"][2] * moments_df["nu"][0] + 6 * moments_df["nu"][1] * pow(moments_df["nu"][0], 2) - 3 * pow(moments_df["nu"][0], 4)) * pow(h, 4)
                    ]

print('\nТаблица 2:')
# Отображение Таблицы 1
moments_df.style.set_properties(**{'text-align': 'center'}).hide(axis='index')


Таблица 2:


### 3. Вычислить выборочные среднее $\overline{x_в}$ и дисперсию $\sigma_в^2$ с помощью стандартной формулы и с помощью условных вариант. Убедиться, что результаты совпадают.

Для вычисления выборочного среднего с помощью стандартной формулы используем формулу: $\overline{x_в} = \frac 1 N \sum x_i$, где $x_i$ - элемент выборки.

Для вычисления выборочного среднего с помощью условных вариант используем формулу: $\overline {x_в} = \nu_1^*h + C$.

In [246]:
mean_x = np.sum(sample_df.nu) / N
mean_x_var = moments_df["nu"][0] * h + C

print(f"Выборочное среднее по стандартной формуле: {mean_x}")
print(f"Выборочное среднее с помощью условных вариант: {mean_x_var}")

Выборочное среднее по стандартной формуле: 454.5811965811966
Выборочное среднее с помощью условных вариант: 462.5


Для вычисления выборочной дисперсии с помощью стандартной формулы используем формулу: $\sigma_в^2 = \frac 1 N \sum (x_i - \overline{x})^2$, где $x_i$ - элемент выборки.

Для вычисления выборочного среднего с помощью условных вариант используем формулу: $\sigma_в^2 = (\nu_2^*-(\nu_1^*)^2)h^2$

In [250]:
sigma_x2 = np.sum((sample_df.nu - mean_x) ** 2) / N
sigma_x_var2 = moments_df["mu"][1]

print(f"Выборочная дисперсия по стандартной формуле: {sigma_x2}")
print(f"Выборочная дисперсия с помощью условных вариант: {sigma_x_var2}")

Выборочная дисперсия по стандартной формуле: 3084.380159251954
Выборочная дисперсия с помощью условных вариант: 3212.0


### 4. Вычислить исправленную выборочную дисперсию $S^2$ и исправленное СКО $S$. Сравнить данные оценки с смещёнными оценками дисперсии. Сделать выводы.

Для вычисления исправленной выборочной дисперсии используется следующая формула: $S^2 = \frac {N} {N-1} \sigma_в$.

Для вычисления исправленного СКО используется следующая формула: $S = \sqrt {S^2}$, $\sigma_в = \sqrt {\sigma^2_в}$

In [248]:
S2 = sigma_x2 * N / (N-1)
S = np.sqrt(S2)
sigma_x_var = np.sqrt(sigma_x_var2)

print(f"Исправленная выборочная дисперсия: {S2}")
print(f"Исправленное СКО выборочной дисперсии: {S}")
print(f"СКО выборочной дисперсии с помощью условных вариант: {sigma_x_var}")

Исправленная выборочная дисперсия: 3110.9696433834365
Исправленное СКО выборочной дисперсии: 55.77606694078955
СКО выборочной дисперсии с помощью условных вариант: 56.674509261219015


Сравнив исправленную выборочную дисперсию, выборочную дисперсию и выборочную дисперсию с помощью условных вариант можно заметить, что неисправленная выборочная дисперсия ближе к выборочной дисперсии, рассчитанной с помощью условных вариант.

Следовательно, исправлять выборочную дисперсию стоило, так как исправление не приводит к заниженным оценкам дисперсии.

### 5. Найти статистическую оценку коэффициентов асимметрии $a_s^*$ и эксцесса $\epsilon_s^*$. Сделать выводы.

Для вычисления статистической оценки коэффициента ассиметрии используется формула: $a_s^* = \frac {\mu_3^*} {s^3}$.

Для вычисления статистической оценки коэффициента эксцессы используется формула: $\epsilon_s^* = \frac {\mu_4^*} {s^4} - 3$.

In [249]:
a_s = moments_df["mu"][2] / pow(S, 3)
eps_s = moments_df["mu"][3] / pow(S, 4) - 3

print(f"Статистическая оценка коэффициента ассиметрии: {a_s}")
print(f"Статистическая оценка коэффициента эксцессы: {eps_s}")

Статистическая оценка коэффициентов ассиметрии: -0.42740320387207753
Статистическая оценка коэффициентов эксцесса: -3.50117852689233


Коэффициенты ассимметрии и эксцессы показывают насколько распределение случайных величин скошено вправо или влево (ассимметрия) от нормального и насколько растянуто вверх или вниз от нормального распределения (эксцесса).

Статистическая оценка коэффициента ассимметрии $a_s^* = -0.427$:
* $a_s^* < 0$ - распределение скошено влево, имеет место правосторонняя ассиметрия с удлиненным правым хвостом;
* $0.25 < |a_s^*| < 0.5$ - ассимметрия умеренная.

Статистическая оценка коэффициента эксцессы $\epsilon_s^* = -3.501$:
* $\epsilon_s^* < 0$ - распределение является более низким и пологим;
* $|\epsilon_s^*| = 3.501$ - распределение более низкое и пологое, чем нормальное.

### 6. Для интервального ряда вычислить моду $M_o^*$, медиану $M_e^*$ и коэффициент вариации $V^*$ заданного распределения. Сделать выводы.

Мода для интервального ряда вычисляется по следующей формуле: $M_o^* = x_0 + \frac {n_M - n_{M-1}} {(n_M - n_{M-1}) + (n_M - n_{M+1})}h$, где $x_0$ - нижняя граница модального интервала (интервал с наибольшей частотой), $n_M$ - частота модального интервала, $n_{M-1}$ - частота интервала предыдущего до модального, $n_{M+1}$ - частота интервала следующего после модального.

In [373]:
#  частота модального интервала
nM = interval_df.max()['n_i']

row_with_nM = interval_df.loc[interval_df['n_i'] == nM]

# нижняя граница модального интервала
x0 = row_with_nM['[x_i, x_i+1)'].to_list()[0][0]
index = row_with_nM['i'].to_list()[0]

nM1 = 0
nM2 = 0
# частоты интервалов до модального и после
if (index == 0) :
    nM2 = interval_df.loc[interval_df['i'] == index + 1]['n_i'].to_list()[0]
elif (index == len(interval_df) - 1) :
    nM1 = interval_df.loc[interval_df['i'] == index - 1]['n_i'].to_list()[0]
else :
    nM2 = interval_df.loc[interval_df['i'] == index + 1]['n_i'].to_list()[0]
    nM1 = interval_df.loc[interval_df['i'] == index - 1]['n_i'].to_list()[0]

M0 = x0 + (nM - nM1) / ((nM - nM1) + (nM - nM2)) * h

print(f"Мода для интервального ряда: {M0}")

Мода для интервального ряда: 447.75


Медиана для интервального ряда вычисляется по следующей формуле: $M_e^* = x_0 + \frac {0.5N - n^k_{m-1}} {n_m} h$, где $N$ - объем выборки, $n_m$ - частота медианного интервала, $n^k_{m-1}$ - накопленная частота предыдущего интервала, $x_0$ - нижняя граница медианного интервала.

In [381]:
# медианный интервал
interval_df_less = interval_df.loc[interval_df['Накопленная n_i'] >= round(interval_df.sum()['n_i']/2)]
row_with_frec = interval_df_less.loc[interval_df_less['i'] == len(interval_df_less)]
ind_row_with_frec = row_with_frec['i'].to_list()[0]

# нижняя граница медианного интервала
x02 = row_with_frec['[x_i, x_i+1)'].to_list()[0][0]

# частота медианного интервала
nm = row_with_frec['n_i'].to_list()[0]

# накопленная частота предыдущего интервала
nm2k = 0
if (ind_row_with_frec != 0) :
  nm2k = interval_df.loc[interval_df['i'] == ind_row_with_frec - 1]['Накопленная n_i'].to_list()[0]

Me = x02 + (0.5 * N - nm2k) * h / nm

print(f"Медиана для интервального ряда: {Me}")

Медиана для интервального ряда: 476.07142857142856


Коэффициент вариации вычисляется по формуле: $V^* = \frac {\sigma_в} {\overline {x_в}} 100%$.

In [386]:
# коэффициент вариации
V = sigma_x_var / mean_x_var * 100

print(f"Коэффициент вариации для интервального ряда: {V} %")

Коэффициент вариации для интервального ряда: 12.25394794837168 %


* Мода $M_0^* = 447.75$ - больше всего элементов выборки имеют значение близкое к $447.75$;
* Медиана $M_e^* = 476.071$ - половина чисел принимает значение меньшее, чем 476.071, а половина большее.
* Коэффициент вариации $V^* = 12.254%$, $V^* < 30\%$ - статистическая совокупность считается однородной. Это означает, что большинство вариант находится недалеко от средней, и найденное значение $\overline {x_в}$ хорошо характеризует центральную тенденцию совокупности.

## Выводы

В ходе выполнения данной практической работы были получены практические навыки нахождения точечных статистических оценок параметров распределения.

Для интервального ряда из практической работы №1 были найдены середины интервалов и накопленные частоты, далее для полученных вариант были вычислены условные варианты. Были вычислены условные эмпирические моменты через условные варианты, и с их помощью вычислены начальные и центральные эмпирические моменты. Корректность вычислений была проверена контрольной суммой, которая дала понять, что вычисления были верны.

Были посчитаны выборочное среднее и дисперсия с помощью стандартных формул и с помощью условных вариант. Статистические оценки, вычисленные по стандартным формулам и с помощью условных вариант совпали.

Была найдена статистическая оценка коэффициентов асимметрии и эксцесса. Имеется правосторонняя асимметрия с удлиненным правым хвостом, эта асимметрия умеренная. Эмпирическое распределение является более низким и пологим относительно нормального распределения. Для интервального ряда была вычислена мода, медиана и коэффициент вариации. Мода $= 447.75$, значит, больше всего элементов выборки имеют значение близкое к $447.75$. Медиана $= 476.071$  - половина чисел принимает значение меньшее, чем $476.071$, а половина большее. Коэффициент вариации $= 12.254$, $V^*<30\%$ - статистическая совокупность считается однородной. Это означает, что большинство вариант находится недалеко от средней, и найденное значение $\overline {x_в}$ хорошо характеризует центральную тенденцию совокупности.